### PR TITLE
Add and remove functionality

### PR DESCRIPTION
**Pull request by Olamarx**
- Add the add and remove functionalities
- Used local storage
- Used class for each file.

### CODE REVIEW

**Review by ShahierNashaat — CHANGES_REQUESTED** [review]
Hi @Olamarx , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/modules/logicToDisplay.js [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+            }
+          });
+          trash.addEventListener('click', () => {
+            UI.deleteTodo(trash);
+            const text = target.parentElement.parentElement.children[0].children[1].value;
+            Store.removeFromLocalStorage(text);
+          });
+        }
+      });
+    }
```
[the remark this answers is not in the document]
Comment (Olamarx) [reply src/modules/logicToDisplay.js:63]: @ShahierNashaat can you please be explicit? I don't understand what you mean.

[the remark this answers is not in the document]
Comment (ShahierNashaat) [reply src/modules/logicToDisplay.js:63]: You should split them into a function for editing the task and a function for deleting the task.

[the remark this answers is not in the document]
Comment (Olamarx) [reply src/modules/logicToDisplay.js:63]: @ShahierNashaat There is no edit function here at all sir.
I have not added the edit function.
That is a task for tomorrow.

[the remark this answers is not in the document]
Comment (ShahierNashaat) [reply src/modules/logicToDisplay.js:63]: Screenshot from 2022-05-31 20-41-34

[the remark this answers is not in the document]
Comment (Olamarx) [reply src/modules/logicToDisplay.js:63]: Oh!!

Thank you!

**Review by zainsadaqat — CHANGES_REQUESTED** [review]
# Required Changes 🔁 
# Hello @Olamarx 


You're doing a great job on your project. We're proud of you. Good job so far! 

There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights
  ✅  Correct use of Github Flow
  ✅  No linter errors
  ✅  Descriptive Pull Request

## Required Changes ♻️

- [x] As requested by the previous Code Reviewer, Kindly add the functionality to remove an item 👍🏻   😃 
Screen Shot 2022-06-01 at 3 43 31 AM

_Check the comments under the review._




## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding! 👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by BenMukebo — APPROVED** [review]
<h1>Status: Approve  :champagne:  :+1: :confetti_ball: 

### **Hi @Olamarx**, 

### **Congratulations! 🎉 Project Approved ✅**

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 


### To Highlight :1st_place_medal: 
  
-  A function for adding a new task is implemented.✔️ 
-  A function for editing task descriptions is implemented.✔️ 
-  A function for deleting a task is implemented. ✔️ 
- Deleting a task updates all remaining items' indexes, so they represent the current list order and are unique.✔️ 
- All changes to the To-Do List should be saved in local storage.✔️ 

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers :clinking_glasses:  and Happy coding!!! :dancers:💻  

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please ping me @BenMukebo when you comment so I can receive the notification.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**